src/main/java/api/kirana/register/transactions/helpers/ReportingHelper.java

```diff
@@ -1,122 +0,0 @@
-package api.kirana.register.transactions.helpers;
-
-import api.kirana.register.transactions.entity.Transactions;
-import api.kirana.register.transactions.models.AggregationResponse;
-import api.kirana.register.transactions.repo.TransactionsDAO;
-import org.springframework.stereotype.Service;
-
-import java.time.DayOfWeek;
-import java.time.LocalDate;
-import java.time.ZoneId;
-import java.util.Date;
-import java.util.List;
-import java.util.Map;
-@Service
-public class ReportingHelper {
-    private TransactionsDAO transactionsDAO;
-
-    public ReportingHelper(TransactionsDAO transactionsDAO) {
-        this.transactionsDAO = transactionsDAO;
-    }
-    public void getWeeklyAggregations(LocalDate start, LocalDate end, Map<String, AggregationResponse> aggregations) {
```

Comment:
This should not be a void method, it should return the aggregations and you can handle the returned aggregations in the calling functions

Reply:
Same for all void methods here